certif/app/components/certification-candidate-in-staging-item.js

```diff
@@ -1,11 +1,83 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
+import { computed } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
 import config from 'pix-certif/config/environment';
 
+const firstOf20thCentury = -2206310961000;
+
 export default class CertificationCandidateInStagingItem extends Component {
 
   isResultRecipientEmailVisible = config.APP.FT_IS_RESULT_RECIPIENT_EMAIL_VISIBLE;
 
+  birthProvinceCodePattern = '^[0-9][A,a,B,b,0-9][0-9]?$'
+
+  @tracked
+  lastNameFocused = false;
+
+  @tracked
+  firstNameFocused = false;
+
+  @tracked
+  birthdateFocused = false;
+
+  @tracked
+  birthCityFocused = false;
+
+  @tracked
+  birthProvinceCodeFocused = false;
+
+  @tracked
+  birthCountryFocused = false;
+
+  @computed(
+    'args.candidateData.{birthCity,birthCountry,birthProvinceCode,birthdate,firstName,lastName}', 'birthProvinceCodePattern', 'isNotThatOld',
+  )
+  get isDisabled() {
+    return !(this.args.candidateData.lastName
+      && this.args.candidateData.firstName
+      && this.isNotThatOld
+      && this.args.candidateData.birthCity
+      && new RegExp(this.birthProvinceCodePattern).test(this.args.candidateData.birthProvinceCode)
+      && this.args.candidateData.birthCountry);
+  }
+
+  get isNotThatOld() {
+    const [year, month, day] = this.args.candidateData.birthdate.split('-');
+    return new Date(year, month, day).getTime() > firstOf20thCentury;
```

Comment:
Ca peut être un peu flou la première fois qu'on lit ça, du coup je propose une petite variable intermédiaire : 
```suggestion
    const [year, month, day] = this.args.candidateData.birthdate.split('-');
    const inputDate = new Date(year, month, day).getTime();
    return inputDate > firstOf20thCentury;
```